dot/network/service_test.go

```diff
@@ -113,6 +111,7 @@ func createTestService(t *testing.T, cfg *Config) (srvc *Service) {
 
 	t.Cleanup(func() {
 		srvc.Stop()
+		time.Sleep(time.Second)
```

Comment:
@noot We should remove this change if the test is passing without it. It will slow down the test suite.